sdcm/tester.py

```diff
@@ -1554,6 +1562,12 @@ def get_cluster_k8s_local_kind_cluster(self):
         else:
             self.monitors = NoMonitorSet()
 
+    def get_cluster_kafka(self):
+        kafka_backend = self.params.get('kafka_backend')
+        if kafka_backend == 'localstack':
```

Comment:
New config option supports more than 1 value for the kafka backend.
So, need to have some logic for handling others. For example - `raise NotImplementedError()`.